src/layout/DefaultLayout.tsx

```diff
@@ -0,0 +1,161 @@
+import { Flex, Heading, Stack } from '@chakra-ui/react';
+import { ReactNode } from 'react';
+import { FiLayout } from 'react-icons/fi';
+import {
+  MdMonitor,
+  MdOutlineCallToAction,
+  MdOutlineDashboard,
+  MdOutlineViewAgenda,
+} from 'react-icons/md';
+
+import { DashboardOptions } from '../styles/components/DashboardOptions';
+
+interface DefaultLayoutProps {
+  children: ReactNode;
+  isActive1?: boolean;
+  isActive2?: boolean;
+  isActive3?: boolean;
+  isActive4?: boolean;
+  isActive5?: boolean;
+}
+
+const DefaultLayout = ({
+  children,
+  isActive1,
+  isActive2,
+  isActive3,
+  isActive4,
+  isActive5,
+}: DefaultLayoutProps) => {
+  const MenuHeadStyle = {
+    borderBottom: '1px solid',
+    width: '340px',
+    height: '60px',
+    borderColor: '#777777',
+  };
+
+  const divStyle = {
+    // textAlign: 'left',
+    width: '100%',
+    display: 'flex',
+    fontSize: 'medium',
+    alignItem: 'center',
+  };
+
+  const buttonTextStyle = {
+    marginLeft: '12px',
+    display: 'flex',
+    marginTop: '4%',
+  };
+
+  const iconStyle = {
+    fontSize: 'xx-large',
+  };
+
+  return (
+    <>
+      <div
+        style={{
+          display: 'flex',
+          fontFamily: 'Overpass ,sans-serif',
+        }}>
+        <div
+          style={{
+            margin: '55px',
+          }}>
+          <Flex
+            align='center'
+            justify='center'
+            style={MenuHeadStyle}>
+            <Heading
+              width={210}
+              height={51}
+              margin='0 auto'
+              textAlign='center'>
+              <div
+                style={{
+                  fontFamily: 'Overpass, sans-serif',
+                  fontSize: 'lg',
+                  color: 'black',
+                }}>
+                Schedula
+              </div>
+            </Heading>
+          </Flex>
+
+          <Stack
+            spacing={7}
+            direction='column'
+            align='left'
+            w={360}
+            marginTop={10}>
+            <DashboardOptions isActive={isActive1}>
```

Comment:
Ao receber informações sobre a página atual, pode-se fazer um teste e saber se o botão deveria estar ativo ou não.

Reply:
Mudei para que quando a variável Active for igual à uma string certa, ascende o botão.